app/src/main/res/values/colors.xml

```diff
@@ -24,15 +24,15 @@
   <!-- AUDIO COMPONENT -->
   <color name="audioComponentBackground">@color/oppiaDarkBlue</color>
   <!-- SUBMIT BUTTON INACTIVE -->
-  <color name="submitButtonInactive">#56000000</color>
+  <color name="submitButtonInactive">#61000000</color>
   <color name="grey">#CCCCCC</color>
   <!-- INPUT LAYOUT EDITTEXT -->
   <color name="blue_shade_50">#502D4A9D</color>
   <color name="blue_shade_40">#402D4A9D</color>
   <color name="blue_shade_20">#202D4A9D</color>
   <color name="blue_shade_60">#602D4A9D</color>
   <color name="grey_shade_10">#102D4A9D</color>
-  <color name="grey_shade_20">#D6D6D6</color>
+  <color name="grey_shade_20">#33999999</color>
```

Comment:
Is this still a 'grey_shade_20' or is it something else now that the value is different? I'm wondering if we should have a contextual name for this, instead, like oppia_disabled_button_background or similar.

Reply:
Changed it to `oppia_disabled_button_background`